rpc/src/module/chain.rs

```diff
@@ -1191,6 +1191,50 @@ pub trait ChainRpc {
     /// ```
     #[rpc(name = "get_block_median_time")]
     fn get_block_median_time(&self, block_hash: H256) -> Result<Option<Timestamp>>;
+
+    /// Returns the median fee rate in recent number of blocks specified by user.
+    ///
+    /// ## Params
+    ///
+    /// * `max_blocks_scan` - the latest x (default: 42) blocks scanned to calculate median fee rate.
+    /// * `min_transaction` - minimal number transactions(default: 42) included in the block.
+    ///
+    /// ## Returns
+    /// * `Result<Capacity>` - the median fee rate(in Shannons).
+    ///
+    /// When the `blocks_to_scan` is greater than `max_blocks_to_scan(default: 42)`, this RPC returns error;
```

Comment:
> `max_blocks_to_scan(default: 42)`

https://github.com/nervosnetwork/ckb/pull/2946/files#diff-a6635561a24fc78c6e32239721e0ca12c38cd35e0674eb3f7a669b54a2cceb0cR1709
This is a hard-coded value, not a default value.